packages/client/scripts/postinstall.js

```diff
@@ -218,10 +218,7 @@ async function createDefaultGeneratedThrowFiles() {
       index: defaultFileConfig,
       edge: defaultFileConfig,
       default: defaultFileConfig,
-      wasm: {
-        js: path.join(__dirname, 'default-wasm.js'),
-        ts: path.join(__dirname, 'default-wasm.d.ts'),
-      },
+      wasm: defaultFileConfig,
```

Comment:
Just curious, why not comment it out completely?

Reply:
The exports map needs it, and no it will default to the same usual error message when your client was not generated.

```
@prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues
```